src/utils/axiosClient.ts

```diff
@@ -1,16 +1,14 @@
-// axios docs https://axios-http.com/docs/intro
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Remove ESLint disabling for '@typescript-eslint/no-explicit-any', and fix any type related issues in the code.